fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -99,6 +109,8 @@ fun PrimaryButton(
     textColor: Color = FirefoxTheme.colors.textActionPrimary,
     backgroundColor: Color = FirefoxTheme.colors.actionPrimary,
     icon: Painter? = null,
+    iconModifier: Modifier = Modifier,
+    buttonEnabled: Boolean = true,
```

Comment:
Can you also add these to the other Buttons (eg Secondary, Destructive, etc)?

Reply:
Done